pkg/service/service.go

```diff
@@ -134,15 +139,29 @@ func NewService(monitorNotify monitorNotify) *Service {
 	if option.Config.EnableHealthCheckNodePort {
 		localHealthServer = healthserver.New()
 	}
-	return &Service{
+	svc := &Service{
 		svcByHash:       map[string]*svcInfo{},
 		svcByID:         map[lb.ID]*svcInfo{},
 		backendRefCount: counter.StringCounter{},
 		backendByHash:   map[string]*lb.Backend{},
 		monitorNotify:   monitorNotify,
 		healthServer:    localHealthServer,
 		lbmap:           &lbmap.LBBPFMap{},
+		clock:           clock.RealClock{},
 	}
+	svc.lastUpdatedTs.Store(svc.clock.Now())
+	return svc
+}
+
+func (s *Service) GetLastUpdatedTs() time.Time {
+	var ts time.Time
+	val := s.lastUpdatedTs.Load()
+	ts = val.(time.Time)
```

Comment:
Nit: return s.lastUpdateTs.Load().(time.Time) ?

Reply:
Will address it and will add the test.